values.yaml

```diff
@@ -132,9 +145,7 @@ runner:
   # of the annotations to apply to the server pods
   annotations: {}
 
-  # Definition of the serviceAccount used to by Waypoint runners. This will
-  # be shared by both the static runner and the on-demand runner that is
-  # initially configured with the chart install.
+  # Definition of the serviceAccount used to by Waypoint static runner. 
```

Comment:
I would expand that the permissions this needs is usually only to launch on-demand runners, so to create pods and such.